### PR TITLE
test sauce labs tests

### PR DESCRIPTION
**Pull request by dgrammatiko**
DO NOT MERGE THIS!!!!!

### CODE REVIEW

**Comment by dgrammatiko** [comment]
B/C break: https://saucelabs.com/blog/update-firefox-tests-before-oct-4-2022
Waiting for https://github.com/karma-runner/karma-sauce-launcher/pull/278

File: .github/workflows/remote-tests.yml [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
-  pull_request:
-    branches:
-      - master
-    types: [closed]
+on: [push, pull_request]
 
 jobs:
   test:
     runs-on: ubuntu-latest
-    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
```
Comment (wilsonge) [inline .github/workflows/remote-tests.yml:12]: /reminder for self to readd this before merging

**Comment by wilsonge** [comment]
I've had to directly commit the workflow stuff here because the tests wouldn't run anymore because of the higher node version required.

**Comment by dgrammatiko** [comment]
Maybe I could try to use a patch and get the Saucelabs tunnel in a working state. I'll have a look when I get home on both these PRs. The delay on both is due to the broken remote tests...

**Comment by dgrammatiko** [comment]
@wilsonge so I forked the karma-sauce-launcher and applied the PR and the remote tests are green again. Let me know if that's ok as a temp solution, till the upstream repo merges the https://github.com/karma-runner/karma-sauce-launcher/pull/278

**Comment by wilsonge** [comment]
https://github.com/karma-runner/karma-sauce-launcher/issues/275#issuecomment-1318593354 Are we able to stay with the original and use this workaround?

If not your fork is fine. We just need to document somewhere for the future in case they are able to magically unlock things.

**Comment by dgrammatiko** [comment]
@wilsonge applied the override, so no more forks. Merge it and let's see if it actually works

**Comment by wilsonge** [comment]
Looks good. Am I merging this one or you going to push to #215 this new part?

**Comment by dgrammatiko** [comment]
Merge this one, I have to review the other again (it's been a while and some pkgs might need another update)

**Pull request merged**